src/popup/components/ConnectWalletForm.tsx

```diff
@@ -223,6 +230,26 @@ export const ConnectWalletForm = ({
     }
   }, [defaultValues.walletAddressUrl, handleWalletAddressUrlChange]);
 
+  if (showConsent) {
+    return (
+      <AutoKeyAddConsent
+        onAccept={() => {
+          autoKeyAddConsent.current = true;
+          // saveValue('autoKeyAddConsent', true);
```

Comment:
Probably related to my comment above - saving this preference.

Reply:
Yeah, I'm not sure if we want to save, but I kept the code here in case we decide to.
Or, maybe we can ask for consent once per wallet address? Like we'll store the wallet address instead of `true` there.
If it's false, we most likely want to ask again, as that's a happier path for user (in case they find adding key manually is too much after seeing that input).

Reply:
CC @RabebOthmani